### ISSUE TITLE
Invite Code not working.

### ISSUE THREAD

**Issue by VINISHVV** [Ylianst/MeshCentral#2673]
Dear Ylian,

Getting below error after entering invite code for agent download. Till yesterday its working fine, assume this is due to recent changes done in yesterday's update.

```
TypeError: Cannot read property 'mesh//1OQhhi8DBst9OI89csiLFOgl6N3IjowdGdSyk7DCG8Y3r0Z0nP$TWVQJLxO1NDJJ' of undefined
    at handleInviteRequest (C:\M-ACCESS\node_modules\meshcentral\webserver.js:1827:70)
    at Layer.handle [as handle_request] (C:\M-ACCESS\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\M-ACCESS\node_modules\express\lib\router\route.js:137:13)
    at Route.dispatch (C:\M-ACCESS\node_modules\express\lib\router\route.js:112:3)
    at Layer.handle [as handle_request] (C:\M-ACCESS\node_modules\express\lib\router\layer.js:95:5)
    at C:\M-ACCESS\node_modules\express\lib\router\index.js:281:22
    at Function.process_params (C:\M-ACCESS\node_modules\express\lib\router\index.js:335:12)
    at next (C:\M-ACCESS\node_modules\express\lib\router\index.js:275:10)
    at C:\M-ACCESS\node_modules\meshcentral\webserver.js:5409:20
    at Layer.handle [as handle_request] 
```

**Comment by Ylianst** [comment]
Oh, I will fix this tomorrow when back to my development computer. For now you can edit `node_modules/meshcentral/webserver.js` and find this line:

```
if ((obj.meshes[i].domain == domain.id) && (parent.meshes[i].deleted == null) && (obj.meshes[i].invite != null) && (obj.meshes[i].invite.codes.indexOf(req.body.inviteCode) >= 0)) {
```

and replace it with this line:

```
if ((obj.meshes[i].domain == domain.id) && (obj.meshes[i].deleted == null) && (obj.meshes[i].invite != null) && (obj.meshes[i].invite.codes.indexOf(req.body.inviteCode) >= 0)) {
```

Note "parent" is replaced with "obj".

**Comment by Ylianst** [comment]
Submitted the fix, will be in MeshCentral v0.8.45.

**Comment by VINISHVV** [comment]
Thanks ylian.
Issue resolved.
